BackEnd/src/main/java/com/javaschool/controllers/BackEndControllers.java

```diff
@@ -0,0 +1,24 @@
+package com.javaschool.controllers;
+
+import com.javaschool.service.BackEndService;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class BackEndControllers {
+
+    private BackEndService backEndService;
+
+    public BackEndControllers(final BackEndService backEndService){
+        this.backEndService=backEndService;
+    }
+
+    @RequestMapping("/Type")
```

Comment:
Just replace as all lower case on `/Type` value, this is just a typo 🤣 
https://restfulapi.net/resource-naming/